### PR TITLE
[CloudPC]guoantang/Add regionGroup into domainJoinConfiguration

### PR DESCRIPTION
**Pull request by Guoan-Tang**

### CODE REVIEW

File: api-reference/beta/resources/cloudpcdomainjoinconfiguration.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Represents a defined configuration of how a provisioned Cloud PC device will be
...
 |type|[cloudPcDomainJoinType](#cloudpcdomainjointype-values)|Specifies how the provisioned Cloud PC will be joined to Azure AD. If you choose the `hybridAzureADJoin` type, only provide a value for the **onPremisesConnectionId** property and leave **regionName** as empty. If you choose the `azureADJoin` type, provide a value for either **onPremisesConnectionId** or **regionName**. The possible values are: `azureADJoin`, `hybridAzureADJoin`, `unknownFutureValue`.|
+|regionGroup|[cloudpcRegionGroup](../resources/cloudpcsupportedregion.md#cloudpcRegionGroup-values)|The logic geographic group this region belongs to. Multiple regions could belong to one region group. Customer could select a regionGroup when provision a CloudPC, the CloudPC will be put under one of the regions under the group based on resource status. e.g. Europe region group contains North Europe and West Europe regions. Possible values are: 'default', 'australia', 'canada', 'usCentral', 'usEast", 'usWest', 'france', 'germany', 'europe', 'unitedKingdom', 'japan', 'asia', 'india', 'southAmerica', 'usGovernment' annd 'usGovrnmentDod'. Read-only.|
```
Comment (RuiHou105) [inline api-reference/beta/resources/cloudpcdomainjoinconfiguration.md:24]: Please update:
[cloudpcRegionGroup] => [cloudPcRegionGroup]
a CloudPC => a Cloud PC
the CloudPC => the Cloud PC

File: api-reference/beta/resources/cloudpcsupportedregion.md [hunk 2]
```diff
@@ -29,6 +29,9 @@ Represents a supported region to establish an Azure network connection for Cloud
 |id|String|The unique identifier for the supported region. Read-only.|
 |regionStatus|[cloudPcSupportedRegionStatus](#cloudpcsupportedregionstatus-values)|The status of the supported region. Possible values are: `available`, `restricted`, `unavailable`, `unknownFutureValue`. Read-only.|
 |supportedSolution|[cloudPcManagementService](../resources/cloudpconpremisesconnection.md#cloudpcmanagementservice-values)|The supported service or solution for the region. The possible values are: `windows365`, `devBox`, `unknownFutureValue`. Read-only.|
+|regionGroup|[cloudpcRegionGroup](#cloudpcRegionGroup-values)|The logic geographic group this region belongs to. Multiple regions could belong to one region group. Customer could select a regionGroup when provision a CloudPC,
```
Comment (RuiHou105) [inline api-reference/beta/resources/cloudpcsupportedregion.md:32]: Please update:
[cloudpcRegionGroup] => [cloudPcRegionGroup]
a CloudPC => a Cloud PC
the CloudPC => the Cloud PC

**Review by RuiHou105 — DISMISSED** [review]
@Guoan-Tang 
1. Please resolve comments in this PR.
2. Please create a roadmap workitem and attach the number in comment. Here is an [example](https://microsoftgraph.visualstudio.com/Roadmap/_workitems/edit/14460/).

**Comment by Guoan-Tang** [comment]
RoadMap: [Feature 15413](https://microsoftgraph.visualstudio.com/Roadmap/_workitems/edit/15413): [DEVICES AND APPS][CLOUD PC] Add regionGroup into domainJoinConfiguration

**Comment by RuiHou105** [comment]
@Guoan-Tang  Please also update the changelog/Microsoft.CloudManagedDesktop.json

File: api-reference/beta/resources/cloudpcsupportedregion.md [hunk 3]
```diff
@@ -29,6 +29,7 @@ Represents a supported region to establish an Azure network connection for Cloud
...
 |regionStatus|[cloudPcSupportedRegionStatus](#cloudpcsupportedregionstatus-values)|The status of the supported region. Possible values are: `available`, `restricted`, `unavailable`, `unknownFutureValue`. Read-only.|
 |supportedSolution|[cloudPcManagementService](../resources/cloudpconpremisesconnection.md#cloudpcmanagementservice-values)|The supported service or solution for the region. The possible values are: `windows365`, `devBox`, `unknownFutureValue`. Read-only.|
+|regionGroup|[cloudPcRegionGroup](#cloudpcregiongroup-values)|The logic geographic group this region belongs to. Multiple regions could belong to one region group. Customer could select a regionGroup when provision a Cloud PC, the Cloud PC will be put under one of the regions under the group based on resource status. e.g. Europe region group contains North Europe and West Europe regions. Possible values are: 'default', 'australia', 'canada', 'usCentral', 'usEast", 'usWest', 'france', 'germany', 'europeUnion', 'unitedKingdom', 'japan', 'asia', 'india', 'southAmerica', 'euap', 'usGovernment', 'usGovernmentDOD', 'unknownFutureValue', 'norway', 'switzerland' and 'southKorea'. Read-only.|
```
Comment (RuiHou105) [inline api-reference/beta/resources/cloudpcsupportedregion.md:32]: You don't need to add this for cloudPcSupportedRegion as it is covered in Beth's doc PR.

**Comment by Danielabom** [comment]
Contacted author to get info.

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 4]
```diff
@@ -1,5 +1,39 @@
...
+              "ChangeType": "Addition",
+              "Description": "Added the `regionGroup` property to the [cloudPcDomainJoinConfiguration](https://docs.microsoft.com/en-us/graph/api/resources/cloudPcDomainJoinConfiguration?view=graph-rest-beta) resource.",
+              "Target": "cloudPcDomainJoinConfiguration"
+            },
+            {
+              "Id": "bf31e677-d3ae-4c1c-8be3-72900113ccd4",
+              "ApiChange": "Property",
+              "ChangedApiName": "regionGroup",
+              "ChangeType": "Addition",
+              "Description": "Added the `regionGroup` property to the [cloudPcSupportedRegion](https://docs.microsoft.com/en-us/graph/api/resources/cloudPcSupportedRegion?view=graph-rest-beta) resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.CloudManagedDesktop.json:26]:
```suggestion
              "Description": "Added the **regionGroup** property to the [cloudPcSupportedRegion](https://docs.microsoft.com/en-us/graph/api/resources/cloudPcSupportedRegion?view=graph-rest-beta) resource.",
```

**Review by Danielabom — APPROVED** [review]
Thank you for working on this PR @Guoan-Tang. PR approved.

File: api-reference/beta/resources/cloudpcdomainjoinconfiguration.md [hunk 5]
```diff
@@ -21,6 +21,7 @@ Represents a defined configuration of how a provisioned Cloud PC device will be
...
 |type|[cloudPcDomainJoinType](#cloudpcdomainjointype-values)|Specifies how the provisioned Cloud PC will be joined to Azure AD. If you choose the `hybridAzureADJoin` type, only provide a value for the **onPremisesConnectionId** property and leave **regionName** as empty. If you choose the `azureADJoin` type, provide a value for either **onPremisesConnectionId** or **regionName**. The possible values are: `azureADJoin`, `hybridAzureADJoin`, `unknownFutureValue`.|
+|regionGroup|[cloudPcRegionGroup](../resources/cloudpcsupportedregion.md#cloudpcregiongroup-values)|The logic geographic group this region belongs to. Multiple regions could belong to one region group. Customer could select a regionGroup when provision a Cloud PC, the Cloud PC will be put under one of the regions under the group based on resource status. e.g. Europe region group contains North Europe and West Europe regions. Possible values are: 'default', 'australia', 'canada', 'usCentral', 'usEast", 'usWest', 'france', 'germany', 'europeUnion', 'unitedKingdom', 'japan', 'asia', 'india', 'southAmerica', 'euap', 'usGovernment', 'usGovernmentDOD', 'unknownFutureValue', 'norway', 'switzerland' and 'southKorea'. Read-only.|
```
Comment (Danielabom) [inline api-reference/beta/resources/cloudpcdomainjoinconfiguration.md:24]:
```suggestion
|regionGroup|[cloudPcRegionGroup](../resources/cloudpcsupportedregion.md#cloudpcregiongroup-values)|The logic geographic group this region belongs to. Multiple regions could belong to one region group. A customer could select a regionGroup when provisioning a Cloud PC, the Cloud PC will be put under one of the regions under the group based on resource status. e.g. Europe region group contains North Europe and West Europe regions. Possible values are: 'default', 'australia', 'canada', 'usCentral', 'usEast", 'usWest', 'france', 'germany', 'europeUnion', 'unitedKingdom', 'japan', 'asia', 'india', 'southAmerica', 'euap', 'usGovernment', 'usGovernmentDOD', 'unknownFutureValue', 'norway', 'switzerland' and 'southKorea'. Read-only.|
```

**Comment by Danielabom** [comment]
@Guoan-Tang, your build failed. Please check, thanks!

**Comment by RuiHou105** [comment]
@Lauragra  Could you please help review this PR? The [failed build](https://dev.azure.com/msrobots/apidoctor/_build/results?buildId=107398&view=logs&j=fd490c07-0b22-5182-fac9-6d67fe1e939b&t=dac0b392-2308-5704-b9f7-c5b20adf012e&l=67) is related to the missing framework. We're not sure how to fix it.

**Comment by Danielabom** [comment]
I asked @millicentachieng to help us escalate this build error.

File: changelog/Microsoft.CloudManagedDesktop.json (hunk 4 above)
Comment (Lauragra) [inline changelog/Microsoft.CloudManagedDesktop.json:26]: This PR does not include an update to the cloudPcSupportedRegion resource topic to add the new property, but we list it in the Changelog. Please reconcile this.

**Comment by Danielabom** [comment]
Hi @Guoan-Tang, were you able to take a look at Laura's comments?

**Comment by Guoan-Tang** [comment]
> Hi @Guoan-Tang, were you able to take a look at Laura's comments?

sure, simplified change log, please review again, thanks!

**Review by Danielabom — APPROVED** [review]
Changes are in place. PR approved.

**Pull request merged**